src/rtc/mosaic_geobursts.py

```diff
@@ -103,22 +103,22 @@ def check_reprojection(geogrid_mosaic,
     return flag_requires_reprojection
 
 
-def weighted_mosaic(list_rtc_images, list_nlooks, geo_filename,
-                    geogrid_in=None, verbose = True):
+def _weighted_mosaic(list_rtc_images, list_nlooks,
```

Comment:
The function name is a bit confusing because we have `weighted_mosaic()` in the following function definition. How about something like `_compute_weighted_mosaic()`?